thirdparty/itt_collector/sea_itt_lib/IttNotifyStdSrc.cpp

```diff
@@ -1577,15 +1577,13 @@ void FillApiList(__itt_api_info* api_list_ptr) {
 #define ITT_STUB_IMPL_ORIG(name) ITT_STUB_IMPL(name)
 #ifdef _DEBUG  // dangerous stub that doesn't return anything (even when expected) but records the function call for
                // statistics sake
-#    define ITT_STUB_NO_IMPL(fn)                                              \
```

Comment:
thirdparty change, it's better to revert
codestyle?

Reply:
did you request a change for your own code? :))

Reply:
yep, just not to forget
no one would have noticed)